pyk/src/pyk/kast/inner.py

```diff
@@ -885,12 +885,14 @@ def collect(callback: Callable[[KInner], None], kinner: KInner) -> None:
         callback: Function with the side effect of collecting desired information at each AST node.
         kinner: The term to traverse.
     """
+    subterms = [kinner]
```

Comment:
I guess the `#TODO` at the top of this function can be removed now?

Reply:
Indeed, good catch.